src/core/DropdownMenu.tsx

```diff
@@ -116,16 +150,16 @@ const Link = ({ url, title, subtitle, iconName, children }: LinkProps) => {
   return (
     <a
       href={url}
-      className="menu-link group block p-8 hover:bg-light-grey hover:text-cool-black rounded"
+      className="menu-link group block p-8 rounded
```

Comment:
The styling of the dropdown menu items needs to be revisited. Looking at the figma design, the paddings and border radii aren't right